docs/guide/user_management.md

```diff
@@ -17,12 +17,33 @@ You can customize it according to your needs.
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
-### Setting password expiration
+## Password settings
```

Comment:
There are also password settings of a sort in the User menu (change password). Maybe a more unambiguous title?

Reply:
How about _Password complexity_?

Reply:
Maybe "Password rules"? So it tries to cover any future policies that may be added.

Reply:
That's a great idea!